<launch>
    <node pkg="tare_planner" type="coverage_boundary_publisher" name="coverage_boundary_publisher" output="screen" ns="sensor_coverage_planner">
        <rosparam command="load" file="$(find tare_planner)/config/coverage_boundary_publisher.yaml" />
    </node>
</launch>